### ISSUE TITLE
Sync with TMF (See LS received 16/07/2024)

### ISSUE THREAD

**Issue by boucadair** [ietf-wg-nmop/draft-ietf-nmop-network-incident-yang#16]
Per: https://mailarchive.ietf.org/arch/msg/nmop/LVZmlNOjDFjx3PqRLrjvTSnWnWg/

> @authors of draft-ietf-nmop-network-incident-yang: Parts of the LS are relevant to the scope of the incident mgt draft. Sharing your thoughts and candidate actions would be appreciated. Thanks.

**Comment by billwuqin** [comment]
@Med
First, it looks the current draft has already aligned with what TMF incident management API profile is doing, see annex A of LS, I agree the mapping between IETF Incident management and TMF API management is correct. It is not clear what else we need to do to make further alignment.
For the terminology, we referenced to both TMF incident terminology and IETF incident terminology work. Do we need to only reference IETF incident terminology work, it could be.

Secondly, I think IETF incident management work focuses on network level incident management, will not be tied with OSS or under the control of OSS, We have made change to reflect in the latest version which hasn't been submitted yet, e.g., single domain close loop, multi-domain close loop scenarios.
